src/style.css

```
@@ -0,0 +1,145 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 11px;
+  font-weight: 400;
+}
+
+.card {
+  background-color: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.card--wrapper {
+  width: 200px;
+  padding: 0 15px;
+}
+
+.card__imac {
+  width: 160px;
+  height: 134px;
+
+  object-fit: cover;
+  object-position: center;
+}
+
+.imac--wrapper {
+  margin: 30px 0 45px;
+}
+
+.card__characteristics {
+  line-height: 15px;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.characteristics--wrapper {
+  margin-bottom: 6px;
+}
+
+.card__code {
+  line-height: 13px;
+
+  align-self: flex-start;
+}
+
+.code--wrapper {
+  margin-bottom: 15px;
+}
+
+.card__rating {
+  width: 100%;
+
+  display: flex;
+  justify-content: space-between;
+}
+
+.rating--wrapper {
+  margin-bottom: 25px;
+}
+
+.card__stars {
+  display: flex;
+  align-items: flex-start;
+}
+
+.card__review {
+  line-height: 13px;
+
+  align-self: center;
+}
+
+.card__star {
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+
+  align-self: flex-end;
+  background-position: left;
+  background-repeat: no-repeat;
+  background-image: url("images/star.svg");
+}
+
+.card__star:nth-child(-n +4) {
+  background-image: url("images/star-active.svg");
+}
+
+.card__price {
+  width: 100%;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.price--wrapper {
+  margin-bottom: 20px;
+
+}
+
+.card__name {
+  line-height: 13px;
+}
+
+.card__grn {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 19px;
+}
+
+.card__buy {
+  width: 100%;
+  padding: 12px 0;
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+  line-height: 16px;
+  color: rgba(255, 255, 255, 1);
```

Нет смысла использовать  rgba, если ты не меняешь прозрачность.
```suggestion
  color: rgb(255, 255, 255);
```